documentation/sos.md

```diff
@@ -19,8 +19,7 @@ Getting a version of lldb that works for your platform can be a problem sometime
 
 ## Using SOS
 
-* [SOS debugging for Linux/MacOS](sos-debugging-extension.md)
-* [SOS debugging for Windows](sos-debugging-extension-windows.md)
+* [SOS debugging](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/sos-debugging-extension)
```

Comment:
```suggestion
* [SOS debugging](https://learn.microsoft.com/dotnet/core/diagnostics/sos-debugging-extension)
```
Removing the English portion in the url, although I'm not sure if its best to link to microsoft learn or the dotnet/docs/core/diagnostics/sos-debugging-extension.md. @tommcdon, do you know?

Reply:
Thank you @mikem8361!